modules/gdscript/tests/scripts/analyzer/features/return_covariant_untyped.out

```diff
@@ -0,0 +1,6 @@
+GDTEST_OK
+>> WARNING
+>> Line: 24
+>> UNSAFE_METHOD_ACCESS
+>> The method "greetings()" is not present on the inferred type "Variant" (but may be present on a subtype).
+Hello Sub Class!
```

Comment:
As expected, this still produces the correct result, however, it shows a warning as it is _technically_ unsafe to do so.